### PR TITLE
FWI-2188 - Update insights-admission plugin to support Pluto

### PR DESCRIPTION
**Pull request by makoscafee**
This PR fixes #
Enables pluto to the admission controller.

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What alternative solution should we consider, if any?

[Internal Ticket FWI-2188](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-2188)

### CODE REVIEW

File: plugins/admission/pkg/admission/handler.go [hunk 1]
```diff
@@ -167,7 +168,15 @@ func processInputYAML(ctx context.Context, configurationObject models.Configurat
...
 	}
 
-	// TODO add Pluto report
+	if configurationObject.Reports.Pluto {
+		logrus.Info("Running Pluto")
+		plutoReport, err := pluto.ProcessPluto(input)
+		if err != nil {
+			return false, nil, nil, err
+		}
+		reports = append(reports, plutoReport)
```
[the remark this answers is not in the document]
Comment (makoscafee) [reply plugins/admission/pkg/admission/handler.go:177]: just realized the version of pluto used(3.4.5) is very old compared `5.x` that is used at the moment. Going to update update it and give it another go.

File: plugins/admission/pkg/pluto/pluto.go [hunk 2]
```diff
@@ -42,9 +63,54 @@ func ProcessPluto(input []byte) (models.ReportInfo, error) {
 	if err != nil {
 		return report, err
 	}
+	_ = apiInstance.DisplayOutput()
```
Comment (makoscafee) [inline plugins/admission/pkg/pluto/pluto.go:66]: why do we need this?

**Review by makoscafee — COMMENTED** [review]
apparently i can not approve this PR since i am the author.

File: plugins/admission/pkg/pluto/pluto.go (hunk 2 above)
Comment (ivanfetch-fw) [reply plugins/admission/pkg/pluto/pluto.go:66]: Ooo - thank you, that was left over troubleshooting to display the Pluto struct in admission logs. Removed.

**Review by ivanfetch-fw — APPROVED** [review]
I'm approving this PR to unblock merging, as @makoscafee  has indicated his approval in a Github comment, and [in Slack](https://fairwindsops.slack.com/archives/CNQBGRGFN/p1656097426398159?thread_ts=1656095808.160069&cid=CNQBGRGFN).

**Pull request merged**